src/dmd/access.d

```diff
@@ -60,7 +60,7 @@ bool checkAccess(AggregateDeclaration ad, Loc loc, Scope* sc, Dsymbol smember)
                 return false;
         }
 
-        ad.error(loc, "member `%s` is not accessible%s", smember.toChars(), (sc.flags & SCOPE.onlysafeaccess) ? " from `@safe` code".ptr : "".ptr);
+        ad.error(loc, "%s `%s` is not accessible%s", smember.kind(), smember.toChars, (sc.flags & SCOPE.onlysafeaccess) ? " from `@safe` code".ptr : "".ptr);
```

Comment:
Side effects must have `()`.

Reply:
You literally just removed the `()` for no good reason.

Reply:
@dandrei279 You may want to submit a PR to fix this.

Reply:
I opened this [0] PR.

[0] https://github.com/dlang/dmd/pull/12959